packages/core/scripts/optimistic-oracle-umip/1_Propose.js

```diff
@@ -0,0 +1,113 @@
+// This script generates and submits an identifier-add upgrade transaction to the DVM. It can be run on a local ganache
+// fork of the main net or can be run directly on the main net to execute the upgrade transactions.
+// To run this on the localhost first fork main net into Ganache with the proposerWallet unlocked as follows:
+// ganache-cli --fork https://mainnet.infura.io/v3/d70106f59aef456c9e5bfbb0c2cc7164 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25
+// Then execute the script as: truffle exec ./scripts/identifier-umip/1_Propose.js --network mainnet-fork --identifier USDETH --identifier ETHBTC from core
```

Comment:
nit: this comment is wrong. This script does not use `identifier`

Reply:
Done.